docs/userguide/parameters.md

```diff
@@ -1,4 +1,205 @@
 # Parameters
 
-This page will soon contain information about parameters.
-In the meantime, please see the [examples](../../examples/examples) instead.
\ No newline at end of file
+Parameters are fundamental for BayBE, as they configure the ```SearchSpace``` and serve 
+as the direct link to the controllable variables in your experiment. 
+Before starting an iterative campaign, the user is required to specify the exact 
+parameters they can control and want to consider in their optimization.
+
+```{note}
+BayBE identifies each parameter by a ```name```. All parameter names in one 
+campaign must be unique.
+```
+
+BayBE distinguishes several parameter types. Because under the hood they need to be 
+treated very differently, we classify parameters in two different families: discrete 
+and continuous parameters.
+
+## Continuous Parameters
+
+### ```NumericalContinuousParameter```
+This is currently the only continuous parameter BayBE supports. 
+This parameters type defines possible values from a numerical interval called 
+```bounds```, and thus has an infinite amount of possibilities. 
+Unless restrained by constraints, BayBE will consider any possible parameter value that 
+lies within the chosen interval.
+
+```python
+from baybe.parameters import NumericalContinuousParameter
+
+NumericalContinuousParameter(
+   name = "Temperature",
+   bounds = (0, 100),
+)
+```
+
+## Discrete Parameters
+A discrete parameter has a finite, countable set of possible values. 
```

Comment:
Finiteness implies countability, so this is redundant.

Reply:
fixed